src/Extensions/AddressExtensions.cs

```diff
@@ -25,5 +27,28 @@ public static string ConvertAddressToTitleCase(this string address)
             result += splitAddress[splitAddress.Length - 1].ToUpper();
             return result;
         }
+
+        public static AddressDetail ConvertStringToObject(this string address)
+        {
+            var addressDetails = address.Split(',');
+
+            return new AddressDetail()
+            {
+                HouseNo = addressDetails[0].Split(' ')[0].Trim(),
+                Street = $"{addressDetails[0].Split(' ')[1].Trim()} {addressDetails[0].Split(' ')[2].Trim()}",
+                Area = addressDetails[1].Trim(),
+                Town = addressDetails[2].Trim(),
+                County = addressDetails[3].Trim(),
+                Postcode = addressDetails[4].Trim()
+            };
```

Comment:
The above method doesn't cover all possible address structures (ie not all addresses have 5 lines, not all streets are 2 words etc)